Framework/CurveFitting/test/Algorithms/FitPowderDiffPeaksTest.h

```diff
@@ -107,8 +436,18 @@ class FitPowderDiffPeaksTest : public CxxTest::TestSuite {
     if (!peakdataws) {
       return;
     }
-
     TS_ASSERT_EQUALS(peakdataws->getNumberHistograms(), 5);
+    TS_ASSERT_EQUALS(peakdataws->blocksize(), 2284);
+    TS_ASSERT_EQUALS(peakdataws->x(0).rawData(), peakdataws->x(1).rawData());
+    TS_ASSERT_EQUALS(peakdataws->x(0).rawData(), peakdataws->x(2).rawData());
+    TS_ASSERT_EQUALS(peakdataws->x(0).rawData(), peakdataws->x(3).rawData());
+    TS_ASSERT_EQUALS(peakdataws->x(0).rawData(), peakdataws->x(4).rawData());
```

Comment:
There is an `operator==` for these types available in `#include "MantidTestHelpers/HistogramDataTestHelper.h"` (it is not included by default since we do not want to do exact vector comparison in general code). But using `rawData` is also fine :+1:.